packages/runtime/test/transport/identityDeletionProcess.test.ts

```diff
@@ -0,0 +1,152 @@
+        });
+    });
+
+    describe(GetIdentityDeletionProcessesUseCase.name, () => {
+        test("should get all Identity deletion processes", async function () {
+            const cancelledIdentityDeletionProcess = (await transportService.identityDeletionProcesses.initiateIdentityDeletionProcess()).value;
+            await transportService.identityDeletionProcesses.cancelIdentityDeletionProcess();
+            const activeIdentityDeletionProcess = (await transportService.identityDeletionProcesses.initiateIdentityDeletionProcess()).value;
+
+            const result = await transportService.identityDeletionProcesses.getIdentityDeletionProcesses();
+            expect(result).toBeSuccessful();
+
+            const identityDeletionProcesses = result.value;
+            expect(identityDeletionProcesses).toHaveLength(2);
+            expect(identityDeletionProcesses[0].id.toString()).toBe(cancelledIdentityDeletionProcess.id.toString());
+            expect(identityDeletionProcesses[0].status).toBe(IdentityDeletionProcessStatus.Cancelled);
+            expect(identityDeletionProcesses[1].id.toString()).toBe(activeIdentityDeletionProcess.id.toString());
+            expect(identityDeletionProcesses[1].status).toBe(IdentityDeletionProcessStatus.Approved);
+        });
+
+        test("should return an empty list trying to get all Identity deletion processes if there are none", async function () {
+            const result = await transportService.identityDeletionProcesses.getIdentityDeletionProcesses();
+            expect(result).toBeSuccessful();
+
+            const identityDeletionProcesses = result.value;
+            expect(identityDeletionProcesses).toHaveLength(0);
+        });
+    });
+
+    describe(CancelIdentityDeletionProcessUseCase.name, () => {
+        test("should cancel an Identity deletion process", async function () {
+            await transportService.identityDeletionProcesses.initiateIdentityDeletionProcess();
+            const result = await transportService.identityDeletionProcesses.cancelIdentityDeletionProcess();
+            expect(result).toBeSuccessful();
+
+            const cancelledIdentityDeletionProcess = result.value;
+            expect(cancelledIdentityDeletionProcess.status).toBe(IdentityDeletionProcessStatus.Cancelled);
+        });
+
+        test("should return an error trying to cancel an Identity deletion process if there is none active", async function () {
+            const result = await transportService.identityDeletionProcesses.cancelIdentityDeletionProcess();
+            expect(result).toBeAnError("No active identity deletion process found.", "error.runtime.identity.noActiveIdentityDeletionProcess");
+        });
+    });
+
+    describe(ApproveIdentityDeletionProcessUseCase.name, () => {
+        test("should approve an waiting Identity deletion process", async function () {
```

Comment:
"a waiting" instead of "an waiting"

Would also avoid talking about a "waiting" IdentityDeletionProcess and instead clarify that it is about an Identity deletion process that is "waiting for approval".